### PR TITLE
Added "all" flag to delete + added function PrepareTrainerJobNameList…

### PR DESCRIPTION
**Pull request by natasharomm**
… to "job_list.go"

### CODE REVIEW

File: cmd/completion/README.md [hunk 1]
```diff
@@ -173,5 +173,5 @@ Or:
 Add to .zshrc:
 
     autoload -U compinit; compinit -i
-    source <(runai compleiton zsh)
+    source <(runai completion zsh)
```
Comment (yodarshafrir1) [inline cmd/completion/README.md:176]: 🥇

File: cmd/job/job_list.go [hunk 2]
```diff
@@ -126,6 +126,24 @@ func PrepareTrainerJobList(kubeClient *client.Client, namespaceInfo types.Namesp
 	return jobs, invalidJobs, nil
 }
 
+func PrepareTrainerJobNameList(kubeClient *client.Client, namespaceInfo types.NamespaceInfo) ([]string, error) {
```
Comment (yodarshafrir1) [inline cmd/job/job_list.go:129]: I think I'd rename it to `ListJobNamesByNamespace`

File: cmd/job/job_list.go [hunk 3]
```diff
@@ -126,6 +126,24 @@ func PrepareTrainerJobList(kubeClient *client.Client, namespaceInfo types.Namesp
...
+	jobs, invalidJobs, err := PrepareTrainerJobList(kubeClient, namespaceInfo)
+	if err != nil {
+		log.Error(err)
+		os.Exit(1)
+	}
+
+	result := make([]string, 0, len(jobs)+len(invalidJobs))
```
Comment (yodarshafrir1) [inline cmd/job/job_list.go:136]: I'd go with:
```
var result []string
```
Instead of using make, but totally up to you

Comment (yodarshafrir1) [inline cmd/job/job_list.go:136]: can you please rename result to: `activeJobs` or `activeJobsByNames`

**Review by yodarshafrir1 — CHANGES_REQUESTED** [review]
Looks great, Please see my notes

File: cmd/job/job_list.go (hunk 2 above)
Comment (natasharomm) [reply cmd/job/job_list.go:129]: :ok_hand:

File: cmd/job/job_list.go (hunk 3 above)
Comment (natasharomm) [reply cmd/job/job_list.go:136]: yeah, thanks! I just copied it from somewhere else, I didn't know the syntax :) will know in time

Comment (natasharomm) [reply cmd/job/job_list.go:136]: 👌

**Review by yodarshafrir1 — APPROVED** [review]
💯

**Pull request merged**